OrbitQt/orbitmainwindow.cpp

```diff
@@ -642,6 +646,8 @@ void OrbitMainWindow::OpenCapture(const std::string& filepath) {
   GOrbitApp->OnLoadCapture(filepath);
   setWindowTitle(QString::fromStdString(filepath));
   ui->MainTabWidget->setCurrentWidget(ui->CaptureTab);
+  ui->actionSave_Capture->setDisabled(false);
```

Comment:
maybe nit, maybe its fine as is. Do we use `setDisabled` in other places instead of `setEnabled` ? 
```suggestion
  ui->actionSave_Capture->setEnabled(true);
```
I think this is a tiny little bit easier to read.

Reply:
I just saw `setDisabled` is used often in this context, so feel free to dismiss